interp/constrintern.ml

```diff
@@ -1915,10 +1915,10 @@ let internalize globalenv env pattern_mode (_, ntnvars as lvar) c =
 	    (merge_impargs l args) loc
 
     | CRecord fs ->
-       let st = Evar_kinds.Define (not (Program.get_proofs_transparency ())) in
```

Comment:
It seems like the `Program.get_proofs_transparency()` should still appear somehow. 
ie the current PR is losing that information.

Reply:
I suppose. How do I add it back?